InnerEye/ML/Histopathology/datamodules/base_module.py

```diff
@@ -46,16 +50,25 @@ def __init__(self, root_path: Path, max_bag_size: int = 0, batch_size: int = 1,
         :param cache_mode: The type of caching to perform, i.e. whether the results of all
         transforms up to the first randomised one should be computed only once and reused in
         subsequent iterations:
-          - `MEMORY`: the entire transformed dataset is kept in memory for fastest access;
-          - `DISK`: each transformed sample is saved to disk and loaded on-demand;
-          - `NONE` (default): no caching is performed.
-        :param save_precache: Whether to pre-cache the entire transformed dataset upfront and save
+          - `MEMORY`: MONAI CacheDataset is used, the entire transformed dataset is kept in memory for fastest access;
+          - `DISK`: MONAI PersistentDataset is used, each transformed sample is saved to disk and loaded on-demand;
+          - `NONE` (default): standard MONAI dataset is used, no caching is performed.
+        :param precache_location: Whether to pre-cache the entire transformed dataset upfront and save
```

Comment:
Let's clarify that this applies only to memory caching (cf. comment on disk caching below). Also, I'd maybe rephrase the description less as a Boolean flag ("Whether [...]").

Reply:
done